Core/Core.xcodeproj/project.pbxproj

```diff
@@ -4098,6 +4099,7 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				ENABLE_BITCODE = NO;
```

Comment:
Are we sure we don't want bitcode in Release mode?

Reply:
Doesn't matter, CoreUITests never gets built in release. And if it did, xcodebuild would also throw a fit about XCTest not having bitcode.

Reply:
Ah, I didn't catch that this was for CoreUITests. Does disabling bitcode make things a bit faster then?

Reply:
No idea, all I know was xcodebuild was failing somehow because it was trying to produce bitcode to run tests. No issues in xcode, of course

Reply:
I changed both release and debug settings because that's what xcode does from the gui by default.